src/App.js

```diff
@@ -8,20 +8,30 @@ import CatNew from './pages/CatNew'
 import CatShow from './pages/CatShow'
 import Home from './pages/Home'
 import NotFound from './pages/NotFound'
-import mockCats from './mockCats'
-
+import cats from './mockCats'
 import './App.css'
 
 export class App extends Component {
```

Comment:
There are extra exports on all of your components and pages.  If you export default at the bottom of the component, you do not need the export before declaring the class. 
```suggestion
class App extends Component {
```